src/TaskList.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import TaskList from './TaskList'
+
+export default {
+  component: TaskList,
+  title: 'TaskList',
+}
+
+export const listAll = () => <TaskList />
```

Comment:
Why did you make this a function here? Shouldn't this be a component?

Or am I forgetting that this is how you run a function component...? 🤔

Reply:
It was rendering the component but due to the dynamic routing it is actually not working. I will refactor it later on.